### ISSUE TITLE
[RC] On the Computational Inefficiency of Large Batch Sizes for Stochastic Gradient Descent [S1en0sRqKm]

### ISSUE THREAD

**Issue by reproducibility-org** [reproducibility-challenge/iclr_2019#12]
| Team Name | Affiliation |
|---|---|
| TheUnreasonableOne | None |

 - Paper: [On the Computational Inefficiency of Large Batch Sizes for Stochastic Gradient Descent](https://openreview.net/pdf?id=S1en0sRqKm)
 - Forum: [Comments](https://openreview.net/forum?id=S1en0sRqKm)

**Comment by harshalmittal4** [comment]
Is this different from [RESIDUAL LEARNING WITHOUT NORMALIZATION
VIA BETTER INITIALIZATION](https://openreview.net/pdf?id=H1gsz30cKX)

**Comment by harshalmittal4** [comment]
Add @koustuvsinha, could you please clarify on this issue, the paper name and the link it points to are different actually.

**Comment by koustuvsinha** [comment]
@harshalmittal4 yes this is weird, the paper name doesn't match the name submitted in the form. Changed the name.
